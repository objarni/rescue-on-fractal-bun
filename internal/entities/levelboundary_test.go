package entities

import (
	approvals "github.com/approvals/go-approval-tests"
	px "github.com/faiface/pixel"
	"objarni/rescue-on-fractal-bun/tests"
	"testing"
)

func init() {
	approvals.UseReporter(tests.ReportWithMeld())
}

func Test_levelBoundaryInitialState(t *testing.T) {
	levelBoundary := MakeLevelBoundary(px.R(0, 1, 2, 3))
	approvals.VerifyString(t, levelBoundary.String()+"\n")
}

func Test_levelBoundaryGfx(t *testing.T) {
	levelBoundary := MakeLevelBoundary(px.R(0, 1, 2, 3))
	approvals.VerifyString(t, levelBoundary.GfxOp(nil).String()+"\n")
}
