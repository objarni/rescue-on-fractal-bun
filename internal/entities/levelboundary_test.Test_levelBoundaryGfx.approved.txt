WinOp Sequence:
  WinOp from ImdOp:
    ImdOp Sequence:
      Rectangle from Vec(0, 1) to Vec(0, 3) thickness 2
